fe/fe-core/src/main/java/com/starrocks/sql/optimizer/base/DistributionSpec.java

```diff
@@ -41,14 +41,10 @@ public static final class PropertyInfo {
         public long tableId = -1;
 
         public List<Long> partitionIds = Lists.newArrayList();
-        // record nullable columns generated by outer join
-        public ColumnRefSet nullableColumns = new ColumnRefSet();
-        // col -> all equivalent join columns
-        // For example, if A.a, B.b and C.c are equivalent, then we have three key value paris
-        // A.a -> [A.a, B.b, C.c]
-        // B.b -> [A.a, B.b, C.c]
-        // C.c -> [A.a, B.b, C.c]
-        public final Map<Integer, ColumnRefSet> joinEquivalentColumns = Maps.newHashMap();
+
+        private DistributionDisjointSet nullStrictDisjointSet = new DistributionDisjointSet();
```

Comment:
Actually not. Because there eixits <=> and =, there requirments are different. For `a<=>b, b=c`, we can only deirved null strict set (a->a, b->a) and null relax set(a->a, b->a, c->b ).